modules/300-prometheus/hooks/create_observability_module_config.go

```diff
@@ -0,0 +1,68 @@
+/*
+Copyright 2024 Flant JSC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package hooks
+
+import (
+	"github.com/flant/addon-operator/pkg/module_manager/go_hook"
+	"github.com/flant/addon-operator/sdk"
+	"github.com/flant/shell-operator/pkg/kube_events_manager/types"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+)
+
+const observabilityMCSnapshotName = "observability_moduleconfig"
+
+var observabilityMCManifest = map[string]interface{}{
+	"apiVersion": "deckhouse.io/v1alpha1",
+	"kind":       "ModuleConfig",
+	"metadata": map[string]interface{}{
+		"name": "observability",
+	},
+	"spec": map[string]interface{}{
+		"enabled": "true",
+	},
+}
+
+var _ = sdk.RegisterFunc(&go_hook.HookConfig{
```

Comment:
There is no need to subscribe to module configuration changes since we can use input.ConfigValues.Exists, as shown [here](https://github.com/deckhouse/deckhouse/blob/e3dda04f9e75ca1d0a2704e5bf52207b0a0c7a99/go_lib/hooks/storage_class_change/hook.go#L199C5-L199C9). This simplifies the logic because we already have the list of enabled modules, making the additional subscription redundant.